vars/main.yml

```diff
@@ -10,13 +10,20 @@ pihole_default_packages_apt:
   - 'whiptail'
   - 'ca-certificates'
   - 'sqlite3'
+  - 'apt-utils'
```

Comment:
https://github.com/juju4/ansible_pihole/actions/runs/5917931388/job/16046740303#step:7:356
```
"stderr": "debconf: delaying package configuration, since apt-utils is not installed\ndebconf: delaying package configuration, since apt-utils is not installed\ndebconf: delaying package configuration, since apt-utils is not installed",
```

Reply:
See other pull request as well :) -- https://github.com/r-pufky/ansible_pihole/pull/21#issuecomment-1724282462 basically just specify that these clearly delineate what/why those packages are for (e.g. molecule testing on docker).